bears/r/FormatRBear.py

```diff
@@ -20,12 +20,12 @@ class FormatRBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         r_keep_comments: bool=True,
+                         r_keep_comments: bool=None,
```

Comment:
you haven't changed this and the comments below :O
so just remove that change, having `r_keep_comments=True` is totally resonable^^